components/drawer/Drawer.razor.cs

```diff
@@ -356,41 +360,32 @@ private string Transform
         private void SetClass()
         {
             var prefixCls = "ant-drawer";
-            this.ClassMapper.Clear()
+            ClassMapper.Clear()
                 .Add(prefixCls)
                 .If($"{prefixCls}-open", () => _isOpen)
                 .If($"{prefixCls}-{Placement}", () => Placement.IsIn("top", "bottom", "right", "left"))
                 .If($"{prefixCls}-rtl", () => RTL)
                 ;
 
-            this.TitleClassMapper.Clear()
+            TitleClassMapper.Clear()
                 .If("ant-drawer-header", () => _title.Value != null)
                 .If("ant-drawer-header-no-title", () => _title.Value == null)
                 ;
         }
 
         protected override void OnInitialized()
         {
-            this._originalPlacement = Placement;
+            _originalPlacement = Placement;
 
             // TODO: remove
-            this.SetClass();
+            SetClass();
 
             base.OnInitialized();
         }
 
         protected override void OnParametersSet()
         {
-            this.SetClass();
-            if (string.IsNullOrEmpty(Placement) && Placement != _originalPlacement)
```

Comment:
This block and line 448 block that was removed didn't do anything when Placement was null or empty string. I tested with a without it and it behaved the same both ways. 

I checked when it was added to the repo and it was from the start and also moved in a refactor to fix a bug. I double checked that bug and it is still fixed with this removed. It was related to multiple drawers open.